### ISSUE TITLE
Int values being truncated after 16 digits

### ISSUE THREAD

**Issue by rosebrt** [qwtel/sqlite-viewer-vscode#24]
I have some 20-digit ID values which always fit within the 8 bytes of an int, but are being truncated in this viewer with 0s after 16 non-zero digits. This kinda makes viewing them useless because I need the exact digits.

**Comment by qwtel** [comment]
Have you tried resizing the column?

**Comment by vchav73** [comment]
I'm seeing this problem as well. I did try to resize the column, but it still shows zeros after the 16th digit. To replicate the problem I ran this in [DBeaver](https://dbeaver.io/):

`
CREATE TABLE IF NOT EXISTS test_table (id INTEGER PRIMARY KEY, big_int INTEGER);
INSERT INTO test_table (big_int) VALUES (9223372036854775807);
`

I got this is DBeaver:
![image](https://user-images.githubusercontent.com/2886165/219502193-204d7243-9bb1-4d38-ac69-58537d150680.png)


I then loaded the same database into sqlite-viewer. I got this:
![image](https://user-images.githubusercontent.com/2886165/219502494-24a61a49-5f96-4dc6-8f28-5bb67ee9c530.png)

Thank you.
